spec/lib/miq_automation_engine/miq_ae_service_model_spec.rb

```diff
@@ -136,4 +136,15 @@ module MiqAeServiceModelSpec
       end.to raise_error(NoMethodError)
     end
   end
+
+  describe MiqAeMethodService::MiqAeServiceStorageProfile do
+    let(:storage_profile) { FactoryGirl.create(:storage_profile) }
+    let(:ae_storage_profile) { MiqAeMethodService::MiqAeServiceStorageProfile.new(storage_profile.id) }
+
+    describe "#taggable" do
+      it "StorageProfile is taggable" do
+        expect(ae_storage_profile.taggable?).to be_truthy
```

Comment:
Per our offline discussion, we'll see Greg M's reply on https://github.com/ManageIQ/manageiq/pull/10798 and if that's +ve, we can fold this test into that.